clio-dataaccess-model/src/test/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/DocumentMock.scala

```diff
@@ -24,17 +26,21 @@ case class DocumentMock(
 
 object DocumentMock extends ModelAutoDerivation {
 
-  def default: DocumentMock = DocumentMock(
-    upsertId = UpsertId.nextId(),
-    entityId = Symbol("1234.the-string"),
-    mockKeyLong = 1234L,
-    mockKeyString = "the-string",
-    mockFileMd5 = Some(Symbol("12345abcdefg")),
-    mockFilePath = Some(URI.create("gs://the-bucket/the-path.file")),
-    mockStringArray = Some(Seq.empty),
-    mockPathArray = Some(Seq.empty),
-    mockDocumentStatus = Some(DocumentStatus.Normal)
-  )
+  def default: DocumentMock = {
+    val long = Random.nextLong()
+    val string = Random.nextString(10)
+    DocumentMock(
+      upsertId = UpsertId.nextId(),
+      entityId = Symbol(s"$long.$string"),
```

Comment:
Updated with randomness so we wouldn't get unintentional merging in our existing recovery unit tests.